modules/publication/php/publication.class.inc

```diff
@@ -104,7 +104,11 @@ class Publication extends \NDB_Menu_Filter_Form
                           'p.Description',
                           'GROUP_CONCAT(DISTINCT pc.Name)', // Collaborators
                           // vv Variables of Interest vv
-                          'GROUP_CONCAT(DISTINCT pt.Name, tn.Test_name)',
+                          // need to use IFNULL since GROUP_CONCAT seems to fail
```

Comment:
Effectively, MySQL can't decide if a `null` value is equal or not equal to any other value including another `null`. So comparison with `null` value generally result in no match